{{cookiecutter.repo_name}}/.travis.yml

```diff
@@ -1,16 +1,32 @@
 # Config file for automatic testing at travis-ci.org
+# This file will be regenerated if you run travis_pypi_setup.py
 
 language: python
 
 python:
-  - "3.4"
-  - "3.3"
-  - "2.7"
-  - "2.6"
-  - "pypy"
+  - TOXENV=py34
+  - TOXENV=py33
+  - TOXENV=py27
+  - TOXENV=py26
+  - TOXENV=pypy
 
 # command to install dependencies, e.g. pip install -r requirements.txt --use-mirrors
-install: pip install -r requirements.txt
+install: pip install -U tox
 
 # command to run tests, e.g. python setup.py test
-script: python setup.py test
+script: tox
+
+{% if cookiecutter.use_pypi_deployment_with_travis != 'n' -%}
```

Comment:
Alright, so I'll change the test in the hook to check with `!= 'y'` too, to ensure consistency.
Namely, we'll only apply this if user doesn't change default answer `y`, and we don't apply in any other case.
Let me know if this doesn't sound good.